ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/events/[eventName]/logs/[eventId]/SkippedFunctionCard.tsx

```diff
@@ -0,0 +1,85 @@
+import Link from 'next/link';
+import { Time } from '@inngest/components/Time';
+import { IconFunction } from '@inngest/components/icons/Function';
+import { IconStatusCancelled } from '@inngest/components/icons/status/Cancelled';
+import { IconStatusPaused } from '@inngest/components/icons/status/Paused';
+import { RiArrowRightSLine } from '@remixicon/react';
+
+import { graphql } from '@/gql';
+import { SkipReason } from '@/gql/graphql';
+import graphqlAPI from '@/queries/graphqlAPI';
+
+const skipReasonStatusIcons = {
+  [SkipReason.None]: IconStatusCancelled,
+  [SkipReason.FunctionPaused]: IconStatusPaused,
+} as const satisfies Record<SkipReason, SVGComponent>;
+
+const skipReasonDescriptions = {
```

Comment:
```suggestion
const skipReasonDescriptions: Record<string, string> = {
```

We should make these map types `Record<string, string>` to ensure that we properly handle new skip reasons. There's a chicken and egg problem where we add a new `SkipReason` in the backend, deploy the backend, and _then_ update the UI. This means we can get into a situation where there's a new `SkipReason` and the UI crashes because we're pulling `undefined` values out of the maps.

You can see this pattern here:
https://github.com/inngest/inngest/blob/cdz/add-history-skips-to-ui/ui/apps/dashboard/src/components/SyncStatusPill.tsx

This pattern was born out of real UI crashes 😅. Ideally we'd deploy the UI before the backend, but our type generation strategy precludes that. We should fix our type generation at some point but I'm not sure when that will be

Reply:
Good call. 

✅ Changed for both this new card and `TriggeredFunctionCard` in 5f50c7320209540f64a63f3818b55e99ed7a7252